packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -482,6 +483,32 @@ export class Vector2 {
         result.y = array[offset + 1];
     }
 
+	/**
+	 * Gets the angle between 2 vectors
+	 * @param vector0 defines 1st vector
+	 * @param vector1 defines 2nd vector
+	 * @param ref the reference to assign the result
+	 * @returns the updated reference
+	 */
+	 public static GetAngleBetweenVectorsToRef(vector0: Vector2, vector1: Vector2, ref: number): number {

```

Comment:
The calculation for GetAngleBetweenVectorsToRef is not correct. I do not believe this nor GetAngleBetweenVectors for Vector2s is necessary